src/main/java/com/mongodb/jdbc/MongoResultDoc.java

```diff
@@ -4,36 +4,20 @@
 import java.util.List;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
+import org.bson.BsonValue;
 
 public class MongoResultDoc {
-    public Boolean emptyResultSet;
-    public List<Column> values;
+    public List<Column> columns;
```

Comment:
MongoResultDoc will now be one of two things:
- The first metadata doc, in which case `columns` is non-null and `values` is null
- The remaining result-set docs, in which case `columns` is null and `values` is non-null